docs/components/search/_spec.scss

```diff
@@ -1,46 +1,46 @@
-@import (reference) "components/search/index";
+@import "components/search/index";
```

Comment:
Change to `@import "components/search/SearchFacade";` to make it clear that we are importing because of the mixin definitions.

Reply:
Updated the `@import` statement.